docs/old-reference-guide/modules/events/pages/event-processors/subscribing.adoc

```diff
@@ -158,7 +158,7 @@ For a specific Event Processor to be a Subscribing instance using a persistent s
 
 The `PersistentStreamMessageSource` requires a `PersistentStreamProperties` to set the initial properties to create the persistent stream. The properties contain:
 
-* `streamName`: The name of the persistent stream.
+* `streamName`: The name of the persistent stream. It's a unique identifier of the connection with Axon Sever. Usage of the same name will overwrite the existing connection.
```

Comment:
I think it would be good if the section before `=== Configuration` would also contain some specifics about stream uniqueness. Don't get me wrong, it should be described on this property too. But it is so important, that having it repeated here, thus mentioned before this section, is merited.